tests/scripts/pulp_deb/test_deb_content.sh

```diff
@@ -59,10 +59,16 @@ expect_succ pulp deb repository content modify \
 --repository "${REPO1_NAME}" \
 --remove-content "[{\"pulp_href\": \"${PACKAGE_HREF}\"}]"
 
-expect_succ pulp deb content upload --file "${DEB_FILENAME}" --relative-path "${DEB_FILENAME}" --repository "${REPO1_NAME}"
+expect_succ pulp deb content upload --file "${DEB_FILENAME}" --relative-path "${DEB_FILENAME}" --repository "${REPO1_NAME}" --distribution "my-dist" --component "my-comp"
 expect_succ pulp deb repository content list --repository "${REPO1_NAME}"
 test "$(echo "${OUTPUT}" | jq -r length)" -eq "1"
 
+VERSION_HREF=$(pulp deb repository version show --repository "${REPO1_NAME}" | jq -r .pulp_href)
+expect_succ pulp deb content --type release_component list --repository-version "${VERSION_HREF}"
+test "$(echo "${OUTPUT}" | jq -r '.[0].distribution')" -eq "my-dist"
+test "$(echo "${OUTPUT}" | jq -r '.[0].component')" -eq "my-comp2"
```

Comment:
I designed this test to fail, but the tests turned green... Are the tests running these scripts at all?!

Reply:
Looks like the new test is finally failing though not exactly as expected.

Reply:
I was able to get the test working (and strategically failing) locally. The output in the failure case is not very helpful, but that is not specific to the new test, and fixing it is outside the scope of this feature.